ansible/playbook.yml

```diff
@@ -0,0 +1,52 @@
+- name: todoapp
+  hosts: franky-host
+  remote_user: ec2-user
+  vars_prompt:
+    - name: secret_token
+      private: no
+    - name: trello_token
+      private: no
+    - name: token_token
+      private: no
+
+  tasks:
+  - name: ensure git is at the latest version
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes # become root user, elevating your priviledge to sudo
+  - name: install python
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+  - name: install poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
```

Comment:
Note that this runs every time, and Ansible reports "changed" for this step.

This can be improved by adding a `creates: /home/ec2-user/.local/bin/poetry`. This tells Ansible to skip the step (and report "ok") if that file already exists.

Doing something similar for the "~/.local/bin/poetry install" step is a little more complicated because you want it to re-run if the dependencies have changed.